package.json

```diff
@@ -8,9 +8,6 @@
     "url": "https://github.com/zowe/vscode-extension-for-zowe"
   },
   "private": true,
-  "workspaces": [
-    "packages/*"
-  ],
```

Comment:
Our deployment workflow uses Lerna, which depends on `workspaces` being defined in package.json because our lerna.json file has "useWorkspaces" set to true.

I think either we'll need to leave the `workspaces` property here, or we could consider migrating from lerna v6 to v7 which removed the "useWorkspaces" setting (see [lerna@7 release notes](https://github.com/lerna/lerna/blob/main/CHANGELOG.md#700-2023-06-08))